Lib/bdb.py

```diff
@@ -349,6 +354,9 @@ def set_continue(self):
             while frame and frame is not self.botframe:
                 del frame.f_trace
                 frame = frame.f_back
+            for frame in self.frame_trace_lines:
+                frame.f_trace_lines = self.frame_trace_lines[frame]
```

Comment:
I think this reads a little easier.

```suggestion
            for frame, prev_trace_lines in self.frame_trace_lines.items():
                frame.f_trace_lines = prev_trace_lines
```